scandeval/benchmarks/abstract/base.py

```diff
@@ -791,15 +797,15 @@ def benchmark(self,
         # Get bootstrap sample indices
         test_bidxs = rng.integers(0, len(test), size=(9, len(test)))
 
-        if framework in ['pytorch']:
+        if framework in ['pytorch', 'jax']:
 
             # Set platform-dependent random seeds
             if framework == 'pytorch':
                 import torch
                 torch.manual_seed(4242)
                 torch.cuda.manual_seed_all(4242)
                 torch.backends.cudnn.benchmark = False
```

Comment:
We still need to set random seeds if `framework=='jax'`, so this shouldn't be conditional on the `framework` variable.

Reply:
That's right! Fixed.